add_npm_package.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+# Run as ./add_npm_package.sh name version strategy
+program_exists() {
+  which "$@" &> /dev/null;
+}
+
+if !(program_exists crudini); then
+  echo "crudini is not installed - installing..."
+  sudo yum install crudini
```

Comment:
You mean print the command to install it & exit? It already prints out the output of yum